### ISSUE TITLE
Empty folder after official uninstaller completion is not detected as a leftover

### ISSUE THREAD

**Issue by DmitriyYukhanov** [Klocman/Bulk-Crap-Uninstaller#162]
Hey, would be nice to detect empty folders which was left after official uninstallation.

For example, I've installed Supersonic Sled NVIDIA demo, then uninstalled it.
Official uninstaller clears the demo folder from the C:\Program Files (x86)\NVIDIA Corporation\NVIDIA Demos, but this parent folder is left empty and it's not detected as a leftover.

Maybe, leftovers search could have a pre-uninstall pass to be able to detect such cases?
Like Analyse app > uninstall it > analyse leftovers with consideration of the initial analysis.

This additional pass could be probably used in some other cases to improve leftovers detection.

**Comment by Klocman** [comment]
Good idea, but it's a bit tricky to implement. At the moment you could get around that by using the "Cleanup program files" tool.
